.github/actions/cw-build-status/tsconfig.json

```diff
@@ -0,0 +1,65 @@
+{
+  "compilerOptions": {
+    // TODO(piraka9011): Remove when actions/toolkit#332 is merged and released
+    "skipLibCheck": true,
+    /* Basic Options */
+    // "incremental": true,                   /* Enable incremental compilation */
```

Comment:
is it best practice to include all the commented out settings? Or should you pare this down to just the ones we actually set. I assume there is documentation elsewhere to let the next person find what they need.

Reply:
I've seen both ways in different repos. I followed the same format of `action-ros-ci` and `setup-ros`.
There is docs so yes, removing it might be better.